server/VERSION

```diff
@@ -1 +1 @@
-0.69.5
+0.69.6
```

Comment:
Historically, we have kept them separate.  The mechanism is already in place so I didn't bother to have two separate `VERSION` files with the same number.

Reply:
Understood.  And, there is power in having two separate version numbers, so that we have the flexibility to re-release one without the other.

However, there is also power in having the numbers synchronized when they are being released at the same time.  Yes, there is the down side of having to understand and explain that certain numbers don't exist, but the need for that is rare, and that cost is small compared to being able to say, "This version goes with that version because they have the same number".

So, I just wanted to be sure that you had the option to sync them up, if we wanted to.

Reply:
And another argument in favor of syncing them is that this change is impacting both lungs.

Reply:
> impacting both lungs.

Double pneumonia?